### PR TITLE
Parameterize camera_pipe number of iterations

### PR DESCRIPTION
**Pull request by ronlieb**
rev2, minus the libraries which were inadvertently in a previous branch.

### CODE REVIEW

**Comment by dsharletg** [comment]
Note my comments in the previous PR: https://github.com/halide/Halide/pull/1208

File: apps/camera_pipe/Makefile [hunk 1]
```diff
@@ -2,6 +2,8 @@ include ../support/Makefile.inc
 
 all: process
 
+TIMING_ITERATIONS=5
```
Comment (dsharletg) [inline apps/camera_pipe/Makefile]: Did you see my other comment regarding using the ?= assignment here? https://github.com/halide/Halide/pull/1208/files#diff-c3c0c197f337f74bffb3236bc841bfc8R5

FYI, I just pulled this down and tested it. It *doesn't* work unless you change it to

    TIMING_ITERATIONS ?= 5

Then, you can change TIMING_ITERATIONS successfully.

Please test changes, even trivial ones like this.

**Comment by ronlieb** [comment]
strange , it did work for me , when i tried it. will change it any way and try it

**Comment by ronlieb** [comment]
ok tested it this time 3 ways. 1) omitting any setting, (observed 5 iterations). Explicitly on make TIMING_ITERATIONS=2 (observed 2), As an environment variable on command line set to 3, (observed 3). The envrionment variable was not tested previously.

**Comment by dsharletg** [comment]
Thanks for checking. Sorry, I thought you were trying to enable setting it via an environment variable the whole time, I didn't realize you were just trying to make it a variable in the Makefile. Still, I think it's better if you can set it from an environment variable :)

**Comment by ronlieb** [comment]
yes, i do like supporting both methods, glad you noticed. it is now improved.

**Pull request merged**